plugins/woocommerce-admin/client/products/product-variation-form.tsx

```diff
@@ -42,6 +51,16 @@ export const ProductVariationForm: React.FC< {
 				</ProductFormTab>
 			</ProductFormLayout>
 			<ProductFormFooter />
+
+			<div className="product-variation-form__navigation">
```

Comment:
Could this be passed as a `className` to `SimpleNavigation` instead of a wrapping div?

Reply:
Nope, its required to positioning the simple-navigation bottom centered. Maybe there is a more sofisticated way of doing this but I could not find it.

Reply:
done